sljit_src/sljitConfigInternal.h

```diff
@@ -241,7 +241,9 @@ extern "C" {
 /* Not required to implement on archs with unified caches. */
 #define SLJIT_CACHE_FLUSH(from, to)
 
-#elif defined __APPLE__
+#elif defined __APPLE__ && \
+      defined __ENVIRONMENT_MAC_OS_X_VERSION_MIN_REQUIRED__ && \
+      __ENVIRONMENT_MAC_OS_X_VERSION_MIN_REQUIRED__ >= 1050
```

Comment:
wouldn't it be better to use the documented API for this with:

```
MAC_OS_X_VERSION_MIN_REQUIRED >= MAC_OS_X_VERSION_10_5
```

Reply:
`MAC_OS_X_VERSION_10_5` is not going to be defined on 10.4, but I can compare to 1050 I guess.
I'll take a look and report back.